### PR TITLE
my day 3 portfolio project

### PR DESCRIPTION
**Pull request by mosidrum**
Day 3 portfolio project with html and css. Below are some feautures 
1. Use flexboxes as break-point,
2. Html tags and css rules  to design a beautiful UI
3. No linters errors  
4. toolbar
5. headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <i class="fa-solid fa-bars"></i>
+    </header>
+    <main>
+        <div class="main">
+            <h1>
+                Hey There. I’m Isaac.
+            </h1>
+            <p class="text-top">
+                I'm a Software Developer
+            </p>
```
Comment (MaganaAsiati) [inline index.html:23]: - Kindly for the subtitle (if existing) use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 

* This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <p class="text-down">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="icons">
+                <img src="./assets/VectorG.png" alt="">
+                <img src="./assets/GroupI.png" alt="">
+                <img src="./assets/VectorH.png" alt="">
+                <img src="./assets/GroupT.png" alt="">
+                <img src="./assets/VectorM.png" alt="">
+            </div>
```
Comment (MaganaAsiati) [inline index.html:33]: - Kindly make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). :+1: 

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @mosidrum , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- [ ] Kindly add the license to your project it will make it looks more professional :+1: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by joel-kalema — APPROVED** [review]
Hi @mosidrum 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**